monitor_frontend/views/events_unnested_table.view.lkml

```diff
@@ -129,6 +129,46 @@ view: +events_unnested_table {
     group_item_label: "Broker Count"
   }
 
+  dimension: event_extra__response_id {
+    description: "Rating given by user for csat survey"
+    type: string
+    sql: SAFE_CAST(`mozfun.map.get_key`(${TABLE}.event_extra, 'response_id') AS STRING) ;;
```

Comment:
`mozfun.map.get_key()` returns the value as a string, so casting is only necessary if you want it to be something other than a string.
```suggestion
    sql: `mozfun.map.get_key`(${TABLE}.event_extra, 'response_id') ;;
```
(ditto for the other new string dimensions below)